index.bs

```diff
@@ -2019,23 +2025,29 @@ interface XRInputSourceEvent : Event {
   constructor(DOMString type, XRInputSourceEventInit eventInitDict);
   [SameObject] readonly attribute XRFrame frame;
   [SameObject] readonly attribute XRInputSource inputSource;
+  readonly attribute double? x;
+  readonly attribute double? y;
```

Comment:
I feel like these new attributes should be slightly better qualified, the way that mouse events have moved on from a simple `x`/`y` to `clikent`/`screen`/`page`/`etc` versions. Given that they're currently only produced from "screen" input sources it seems like `screenX`/`screenY` would be appropriate, but I'm very open to other suggestions.